CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.2.0] - 2024-10-16
+### Changed
+- Updated all dependency versions for hive, hadoop, datanucleus, reflections, guava, tez, mockito, junit and added them in pom.xml <properties> section
```

Comment:
Can you please put the "before" and "after" values like we do in the lines below - "Update foo from `1.2.3` to `2.5.0`" for example.

Reply:
Done